infra/Pulumi.kitsuneos.yaml

```
@@ -1,3 +1,5 @@
 config:
+  kitsuneos:deploySite: "true"
   kitsuneos:deploySiteCdn: "false"
   kitsuneos:deployApp: "true"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)